server/MainServer.js

```diff
@@ -0,0 +1,40 @@
+class MainServer {
+    constructor() {
+        this.activeOrder = {};
+        this.activeDriver = {};
+    }
+
+    addOrderToDict(order, gs) {
```

Comment:
make a process order function that takes these parameters, then when we check the status, if it is valid then we can just call the add to dict function